src/components/Button.jsx

```diff
@@ -57,7 +57,13 @@ class Button extends React.Component {
         onPress={onPress}
         accessibilityLabel={accessibilityLabel}
       >
-        <Text style={[styles.text, main && styles.textMain, disabled && styles.textDisabled]}>
+        <Text
+          style={[
+            styles.text,
+            main && styles.textMain,
+            disabled && styles.textDisabled,
+          ]}
+        >
```

Comment:
Sorry about that one it was more of my visualization thing it wasn't actually meant to be commited and I think that was on my first day nothing actually got changed at all its the same code.

Reply:
If you didn't worked on it then leave it how it was, otherwise when this PR is commited it will conflict with any change made in that button, and that lines will remain associated with you name, even if you just pressed some enters in there.